cmd/kubeadm/app/apis/kubeadm/bootstraptokenhelpers_test.go

```diff
@@ -64,14 +64,16 @@ func TestToSecret(t *testing.T) {
 		},
 	}
 	for _, rt := range tests {
-		actual := rt.bt.ToSecret()
-		if !reflect.DeepEqual(actual, rt.secret) {
-			t.Errorf(
-				"failed BootstrapToken.ToSecret():\n\texpected: %v\n\t  actual: %v",
-				rt.secret,
-				actual,
-			)
-		}
+		t.Run(rt.bt.Token.ID, func(t *testing.T) {
```

Comment:
why is this kubeadm change part of this PR?
is this a rebase problem?

Reply:
Yes, it was a rebase concern which is now resolved. Thanks for the lead.